src/components/paradox/results/personality/career/modal/style.scss

```diff
@@ -217,3 +217,50 @@
   .description { font-size: $font-size-sm; }
   .title { font-weight: 500; }
 }
+
+.job {
```

Comment:
Some general css comments:
- Try to list properties alphabetically
- Use the theme mixin for any colors

Reply:
Ok Alphabetized and now using theme for colors in .jobs!